Makefile

```diff
@@ -1,9 +1,10 @@
 CONVERT=npx js-yaml
-SYNTAXES=haskell cabal literateHaskell
+VSCE=npx vsce
```

Comment:
I added `npx` to several commands so that they should work without installing things globally or changing the path